tasks/python-reference-task/README.md

```diff
@@ -0,0 +1,29 @@
+# @cumulus/python-reference-task
+
+This 'task' is a reference task that is included with Cumulus to allow integration testing of the [`Cumulus message adapter`](https://github.com/nasa/cumulus-message-adapter) with a built python lambda.
```

Comment:
```suggestion
This 'task' is a reference task that is included with Cumulus to allow integration testing of the [`cumulus-message-adapter`](https://github.com/nasa/cumulus-message-adapter) with a built python lambda.
```

Reply:
Will update, but worth noting in the original review you suggested [Cumulus message adapter].  I like this better for the links, but without the tics - reason being the tics obscure the link.

Reply:
Updated, made links/tics more consistent fc63e472d

Reply:
My bad. Yeah, I like this better